.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -488,6 +488,7 @@ private double GetIntValue(string intStr)
             var isRoundBefore = false;
             long roundBefore = -1, roundDefault = 1;
             var isNegative = false;
+            var isPreDigit = false;
```

Comment:
This flag indicates previous digits have been processed, right? I'd suggest renaming it to something like isMultiDigit